### PR TITLE
Multiqc findability

### PR DESCRIPTION
**Pull request by nomadscientist**
Put note about the MultiQC tool sometimes not being findable

### CODE REVIEW

**Comment by hexylena** [comment]
Hey @nomadscientist, this might also be resolved by using the new GTN in Galaxy option? 

https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links

If the whole tutorial is converted it is especially helpful. We're really recommending this because the students can access the GTN and Galaxy with a single monitor which is nice for them.

File: topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md [hunk 1]
```diff
@@ -284,7 +284,7 @@ Let us investigate the output log. This type of quality control is essential in
 
 > ### {% icon hands_on %} Hands-on
 >
-> **MultiQC** {% icon tool %} with the following parameters:
+> **MultiQC** {% icon tool %} with the following parameters (please note that sometimes MultiQC doesn't appear from using the search box. You'll find it under FASTQ Quality Control or a similar category):
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md:287]:
```suggestion
> - {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy0) %} with the following parameters:
```

**Comment by nomadscientist** [comment]
Great point! Yes, I'll put that on my list to do!

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Awesome, thanks so much @nomadscientist. I'm really hopeful it helps with tool findability since it's just *so* bad.

**Pull request merged**